### PR TITLE
Apply default params

### PR DESCRIPTION
**Pull request by smacker**
Fix: #113 
Based on: #111 

Update is here: https://github.com/src-d/gemini/commit/e83e750a83d392e569acb625ac354dd2312d332d
Small script to calculate hashtables params: https://github.com/src-d/gemini/commit/311afe001f1c38848751d9a8dff7b0437d59b23b
Fixtures are also updated using new defaults.

### CODE REVIEW

**Comment by bzz** [comment]
Could you check if `./sbt scalastyle` and `make lint-python` pass as well?

Also, could you please document a bit, how `calculate_hashtables/main.py` is intended to be used?  https://github.com/src-d/gemini#dev could be a good place

File: src/main/python/calculate_hashtables/main.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+"""
+Calculate number of hashtabels and size of a band
+using the threshold and sample size of wmh
+"""
```
Comment (bzz) [inline src/main/python/calculate_hashtables/main.py]: If this code is similar or comes from another codebase, could you please add a permlink URL in here as well?

I.e https://github.com/src-d/apollo/blob/bbb9f83fffc93e791fa59f064efadefa270f400a/apollo/hasher.py#L20

Comment (bzz) [reply src/main/python/calculate_hashtables/main.py]: As this code seems to come from [MIT-licensed project](https://github.com/ekzhu/datasketch/blob/master/LICENSE), so you also need to update https://github.com/src-d/gemini/blob/master/LICENSE#L677 by adding a new [section for MIT](https://github.com/apache/spark/blob/master/LICENSE#L276) and then as before - add a link to both, original project and a path to `main.py` in our codebase that contains this code.

**Comment by smacker** [comment]
Not sure we should document this script in readme. It calculates hard-coded params, so it meant to be used only internally. Docstring in the file itself updated.

**Comment by bzz** [comment]
Got it. Internally though, could you help understanding, in what case one would want to run it?

**Comment by smacker** [comment]
We may want to rerun it if ML team provide us new hyper-params.

**Pull request merged**